tms-ui/src/main/frontend/src/app/component/header/project/AddTask.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
/src/app/component/header/project/AddTask.js
please fix packege naming
header is redudant here